auth/dao/auth-dao.go

```diff
@@ -14,6 +14,16 @@ type DAO struct {
 	DB *sql.DB
 }
 
+// AuthCreateRequest contains the information required to create a new authorized user
+type AuthCreateRequest struct {
+	Email    string `valid:"email,required"`
+	Password string `valid:"type(string),required,stringlength(8|64)"`
```

Comment:
Similarly here.

In my opinion we don't allow them to customise it, but we make sure we have some literature to justify our choices. If they wanna change it they can alter the code.

Reply:
Sounds good. I'll address this in a follow up PR that focusses on setting these limits to something other than arbitrary values.